src/vue-froala.js

```diff
@@ -93,7 +93,12 @@ export default (Vue, Options = {}) => {
         this.currentConfig = this.config || this.defaultConfig;
 
         this._$element = jQuery(this.$el);
-
+        
+        if(this._props.value){
+          this._$element[0].innerHTML=this._props.value;
```

Comment:
Doesn't look right. You are setting the HTML although it is already set in https://github.com/froala/vue-froala-wysiwyg/blob/master/src/vue-froala.js#L124

Reply:
not able to set char counter at that moment, thats why used this method to initialize the char counter.